bundles/binding/org.openhab.binding.zwave/database/remotec/zts110.xml

```diff
@@ -0,0 +1,143 @@
+<Product>
+	<Model>ZTS-110</Model>
+	<Label lang="en">Remotec Z-Wave Touchscreen Programmable Thermostat</Label>
+	<CommandClasses>                  <!-- Z-Wave Supported Command Classes: -->
+		<Class><id>0x20</id></Class>  <!-- COMMAND_CLASS_BASIC -->
+		<Class><id>0x27</id></Class>  <!-- COMMAND_CLASS_SWITCH_ALL -->
+		<Class><id>0x31</id></Class>  <!-- COMMAND_CLASS_SENSOR_MULTILEVEL -->
+		<Class><id>0x40</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_MODE -->
+		<Class><id>0x42</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_OPERATING_STATE -->
+		<Class><id>0x43</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_SETPOINT -->
+		<Class><id>0x44</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_FAN_MODE -->
+		<Class><id>0x45</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_FAN_STATE -->
+		<Class><id>0x47</id></Class>  <!-- COMMAND_CLASS_THERMOSTAT_SETBACK -->
+		<Class><id>0x72</id></Class>  <!-- COMMAND_CLASS_MANUFACTURER_SPECIFIC -->
+		<Class><id>0x80</id></Class>  <!-- COMMAND_CLASS_BATTERY -->
+		<Class><id>0x81</id></Class>  <!-- COMMAND_CLASS_CLOCK -->
+		<Class><id>0x85</id></Class>  <!-- COMMAND_CLASS_ASSOCIATION -->
+		<Class><id>0x86</id></Class>  <!-- COMMAND_CLASS_VERSION -->
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>int</Type>
+			<Default>2</Default>
+			<Minimum>1</Minimum>
+			<Maximum>4</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Swing</Label>
+			<Help lang="en"><![CDATA[Set Swing value from 1F/.5C to 4F/2.0C Available settings: 1 - 4]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>int</Type>
+			<Default>2</Default>
+			<Minimum>1</Minimum>
+			<Maximum>4</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Differntial</Label>
+			<Help lang="en"><![CDATA[Set Differential value from 1F/.5C to 4F/2.0C Available settings: 1 - 4]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>int</Type>
+			<Default>500</Default>
+			<Minimum>500</Minimum>
+			<Maximum>4000</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Set Filter Counter</Label>
+			<Help lang="en"><![CDATA[Set filter replacement timer from 500 to 4000 hours.]]> </Help>
+		</Parameter>
+	    <Parameter>
+			<Index>4</Index>
+			<Type>int</Type>
+			
+			<Minimum>0</Minimum>
+			<Maximum>9999</Maximum>
+			<Size>2</Size>
+			<Label lang="en">Report Filter Counter</Label>
+			<Help lang="en"><![CDATA[READ ONLY: Data in hours]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>5</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Scale of Temperature</Label>
+			<Help lang="en"><![CDATA[Set Temperature Scale to Farhenheit or Centigrade. DEFAULT =1 Farhenheit]]> </Help>
+		</Parameter>
+		<Parameter>
+			<Index>6</Index>
+			<Type>int</Type>
+			<Default>1</Default>
+			<Minimum>0</Minimum>
+			<Maximum>1</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Upper Limit of Setpoint A</Label>
```

Comment:
I would just set the min and max to be the min/max alllowed of the C and F, so the smallest in C, and the largest in F. Same for both 6 and 7.